src/features/profile/pages/NotificationSettings/NotificationsSettings.native.test.tsx

```diff
@@ -90,12 +96,78 @@ describe('NotificationSettings', () => {
     mockUseAuthContext.mockReturnValueOnce(baseAuthContext)
     render(<NotificationsSettings />)
 
+    const toggleEmail = screen.getByTestId('Interrupteur Autoriser l’envoi d’e-mails')
+    fireEvent.press(toggleEmail)
+
     const toggleSwitch = screen.getByTestId('Interrupteur Cinéma')
     fireEvent.press(toggleSwitch)
 
     expect(screen.getByTestId('Interrupteur Cinéma')).toHaveAccessibilityState({ checked: true })
   })
 
+  it('should disabled all thematic toggles when email toggle and push toggle are inactive', () => {
+    mockUseAuthContext.mockReturnValueOnce(baseAuthContext)
+    render(<NotificationsSettings />)
+
+    expect(screen.getByTestId('Interrupteur Cinéma')).toBeDisabled()
+    expect(screen.getByTestId('Interrupteur Lecture')).toBeDisabled()
+    expect(screen.getByTestId('Interrupteur Musique')).toBeDisabled()
+    expect(screen.getByTestId('Interrupteur Spectacles')).toBeDisabled()
+    expect(screen.getByTestId('Interrupteur Visites et sorties')).toBeDisabled()
+    expect(screen.getByTestId('Interrupteur Cours et Ateliers')).toBeDisabled()
+  })
+
+  it('should display help message when the email toggle is inactive and user is logged in', () => {
+    mockUseAuthContext.mockReturnValueOnce(baseAuthContext)
+    render(<NotificationsSettings />)
+
+    expect(
+      screen.queryByText(
+        'Tu continueras à recevoir par e-mail des informations essentielles concernant ton compte.'
+      )
+    ).toBeOnTheScreen()
+  })
+
+  it('should display info banner when email and push toggles are inactive', () => {
+    mockUseAuthContext.mockReturnValueOnce(baseAuthContext)
+    render(<NotificationsSettings />)
+
+    expect(
+      screen.queryByText(
```

Comment:
D'après `doc/development/tests/unit-test/general-guidelines.md`, on préfère utiliser `getByText` pour vérifier qu'un élément existe :
```suggestion
      screen.getByText(
```

Reply:
Oups j'ai laissé copilot un peu trop me guider, merci !